docs/dev/development.md

```diff
@@ -48,9 +48,12 @@ To launch the Cloudbreak application execute the `com.sequenceiq.cloudbreak.Clou
 -Dcb.db.port.5432.tcp.port=5432
 -Dcb.identity.server.url=http://192.168.59.103:8089
 -Dserver.port=9091
+-Dcb.schema.migration.auto=true
+-Dcb.schema.scripts.location=FULL_PATH_OF_THE_DATABASE_MIGRATION_SCRIPTS
 ```
 
-The `-Dcb.cert.dir=FULL_PATH_OF_THE_CERTS_DIR_GENERATED_BY_CBD` value above needs to be replaced with the full path of `certs` directory generated by Cloudbreak Deployer e.g. `-Dcb.cert.dir=/Users/myusername/prj/cbd-local/certs`
+The `-Dcb.cert.dir=FULL_PATH_OF_THE_CERTS_DIR_GENERATED_BY_CBD` value above needs to be replaced with the full path of `certs` directory generated by Cloudbreak Deployer e.g. `-Dcb.cert.dir=/Users/myusername/prj/cbd-local/certs`.
+The databse migration is ran automatically by Cloudbreak, this automated migration could be turn off through the `-Dcb.schema.migration.auto=false` VM option. The migration process requires the full path of the migration scripts and it could be specified through the `-Dcb.schema.scripts.location=FULL_PATH_OF_THE_DATABASE_MIGRATION_SCRIPTS` VM option. (The migration scripts are located in the repository of Cloudbreak.)
```

Comment:
I think it does not make sense to specify the cb.schema.scripts.location parameter? Basically the cloudbreak can figure out what is the location, because it is always the same src/main/resources/schema  I believe it can be set as default, because if it is started by cbd then the cbd sets this value anyway.